samples/client/petstore/scala-sttp/src/main/scala/org/openapitools/client/api/PetApi.scala

```diff
@@ -33,7 +33,7 @@ class PetApi(baseUrl: String) {
    * @param pet Pet object that needs to be added to the store
    */
   def addPet(pet: Pet
-): Request[Either[ResponseError[Exception], Pet], Nothing] =
+): Request[Either[ResponseException[String, Exception], Pet], Nothing] =
```

Comment:
@borissmidt thanks for the PR. Is this a breaking change (from `ResponseError` to `ResponseException`) ?

In other words, users need to update their code accordingly after this change, right?

Reply:
I will add a new parameter to check which sttp version is selected and branch depending on that to keep backward compatiblity.

Reply:
Maybe simply target this change to 6.0.x branch as we'll release 6.0.0 in 2022 Q1

Reply:
Oke good idea!

Reply:
I rebased it on the 6.0.x branch